src/pages/account-restore.tsx

```diff
@@ -0,0 +1,129 @@
+import { useStore } from '../store'
+import avatarDefault from '../assets/imgs/avatar.svg?url'
+import checkMarkBlue from '../assets/imgs/checkMarkBlue.svg?url'
+import cancel from '../assets/imgs/cancel.svg?url'
+import { ACCOUNT } from '../routes'
+import { UserCreateStop } from './modals/user-create-stop'
+import { RouteComponentProps } from '@reach/router'
+import { useState } from 'preact/hooks'
+import { Profile } from '../types/profile'
+import { JSXInternal } from 'preact/src/jsx'
```

Comment:
Nitpicky, but might be good to organize these imports a bit. Maybe separate the external dependencies, local components, assets and types?

Also for `RouteComponentProps`, `JSXInternal` and `Profile` it could be `import type`.